packages/grafana-ui/src/components/BrowserLabel/Label.tsx

```diff
@@ -75,11 +96,11 @@ Label.displayName = 'Label';
 
 const getLabelStyles = (theme: GrafanaTheme2) => ({
   base: css`
+    display: inline-block;
```

Comment:
I think with styles changes the space between selected labels is gone. Is it intended?

| before | after |
|--------|------|
| <img width="187" alt="Screen Shot 2021-07-21 at 10 03 00" src="https://user-images.githubusercontent.com/745532/126455247-1caeafa3-022a-42ab-8aeb-46e49a9dab1b.png"> | <img width="208" alt="Screen Shot 2021-07-21 at 10 03 34" src="https://user-images.githubusercontent.com/745532/126455280-91799994-2969-4508-bf1a-50e03db513c9.png"> |

Reply:
Nice catch, I'll fix that. I changed it to inline-block because when the Label is inline, it looks like this when searching. I'll try to get both cases looking good. 
![image](https://user-images.githubusercontent.com/12838032/126496467-b9a6cf6a-c78b-4d1a-a4e1-b2d34c3c2bea.png)

Reply:
Maybe a little hacky, but I increased the itemSize for the FixedSizeList for each loki and prometheus browsers. Because FixedSizeList absolutely positions each label, adjusting the margin didn't help. 
![image](https://user-images.githubusercontent.com/12838032/126511991-24f5d6b0-99bc-4a12-ab13-559b4546fabf.png)